engine/src/main/resources/assets/skins/redTextColor.skin

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Also, consider using a semantic naming such as "warning" instead of red, since red text color is just one way of expressing the situation.